source/2019-11-29-the-ember-times-issue-126.md

```diff
@@ -10,6 +10,7 @@ responsive: true
 Represent Ember in the State of JavaScript 2019 survey 📃,
 Give thanks to Octane Guides contributors 🙏,
 Ember + Tailwind CSS tips and tricks 🎨,
+Ember just got more real with RealWorld 🌟,
```

Comment:
The Alex lint is failing because of the word just. You may want to try adding an exception, maybe before line 10 (Represent Ember)?

Reply:
Yeah, I think it would be great to skip the word then to make the build pass :+1: